### ISSUE TITLE
[1.10.2] Remaining IC2/Tech Reborn Issues still outstanding

### ISSUE THREAD

**Issue by ChubbyTurtle101** [TechReborn/TechReborn#875]
A couple of remaining IC2/Tech Reborn Issues with duplicates removed:
    # This **attempts** to fully **integrate** TR with ic2 recipes (Beta)  <- Also a couple of mispellings corrected here.
    B:"Remove Duplicates when IC2 is installed"=true

1)  Crafting 8 iron plates in square to make IC2:Basic Machine Casing is making TechReborn:Machine Block.  There is no way to make the IC2:Basic Machine Casing which is required to make Tech Reborn Machines with the above config setting.

2) IC2:Sticky Resin (Able to get from IC2 Rubber Trees with Tech Reborn Tap) Does not work in Tech Reborn Extractor.  It should make 3 IC2 Rubber.

Thanks for a great mod and I hope you continue the 1.10.2 back ports as there are quite a few mods that have not moved to 1.11.2.

**Comment by modmuss50** [comment]
Ok thanks for letting me know about the issues. They should be an easy fix.

**Comment by ChubbyTurtle101** [comment]
I noticed that the TechReborn Machine Block is used to make Standard Machine Casing.  Also the Machine Block is used in the Advanced Machine Block which is used for Reinforced Machine Casing.  Maybe an easy temporary solution would to have a TechReborn:Machine Block =  IC2:Basic Machine Casing and visa versa.  This will allow all crafting of all recipes and since both use 8 iron plates, there is no cheating.  I used CraftTweaker to add the above recipe so I can progress.  Thanks.
